### ISSUE TITLE
Migrate From Old Website

### ISSUE THREAD

**Issue by Gerseneck** [frc461/website-2024#13]
Migrate webpages over from our old websites.

Create seed data for the the other pages (newsletter, team history, etc.) after #2 is done.

- [x] check if the home page of the category exists before linking
- [x] migrate rest of the views
- [x] error pages
- [x] add footer

**Comment by ez500** [comment]
Especially if revamping resources section will take a while, I suggest migrating the resources section from the old website as the issue suggests and then revamp it in another issue.

**Comment by Gerseneck** [comment]
> Especially if revamping resources section will take a while, I suggest migrating the resources section from the old website as the issue suggests and then revamp it in another issue.

This won't be an issue, the resources section will be non-static (therefore we can edit it using the actual UI).
